src/modules/ekf2/EKF/covariance.cpp

```diff
@@ -54,7 +54,7 @@ void Ekf::initialiseCovariance()
 {
 	P.zero();
 
-	resetQuatCov((_params.mag_fusion_type != MagFuseType::NONE) ?_params.mag_heading_noise : NAN);
+	resetQuatCov();
```

Comment:
@dagar I made a separate commit for the actual change (this isn't refactoring anymore)